app/controllers/drivers_controller.rb

```diff
@@ -0,0 +1,55 @@
+class DriversController < ApplicationController
+  def index
+    @drivers = Driver.all
+    if params[:search]
```

Comment:
Nice job using the same route and pivoting on the `params` to determine which logic to execute